R/cloud.R

```diff
@@ -112,7 +112,9 @@ cloud_fetch_results <- function(job_id = cloud_job(), pkg = ".") {
 
   out_dir <- file.path(cloud, job_id)
 
-  dir.create(out_dir, showWarnings = FALSE, recursive = TRUE)
+  if (!dir.exists(out_dir)) {
```

Comment:
This has a potential race condition if the directory is created between the check and `dir.create()`, you can just use `dir.create()` unconditionally, if the directory already exists it will fail silently, since we have `showWarnings = FALSE`.